docs/modules/behaviors/DATA_BEHAVIOR.md

```diff
@@ -808,4 +824,23 @@ successEventPayload = {
   * "behaviors.behaviorAlias:[event name]" - arbitrary event triggered by another behavior on this view.
   * "behaviors.behaviorAlias.[property name]:[event name]" - arbitrary event triggered by another behavior's property on this view.
   * "behaviors.behaviorAlias.data:[event name]" - arbitrary event triggered by another behavior's data property on this view (specific example of `behaviorAlias.[property name]:[event name]`).
-  * { '[event name]': < object (or function returning an object) that the event is triggered on > } - arbitrary "event" triggered on the supplied object.
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Merge conflicts

Reply:
they seem to have been fixed along the way already.